admin/horace_install.m

```diff
@@ -1,40 +1,41 @@
 function horace_install()
 %HORACE_INSTALL install this instance of Horace
 %
-HORACE_ON_SUB_PATTERN = '${Horace_CORE}';
-HERBERT_ON_SUB_PATTERN = '${Herbert_CORE}';
+HORACE_ON_PLACEHOLDER = '${Horace_CORE}';
+HERBERT_ON_PLACEHOLDER = '${Herbert_CORE}';
 
 install_root = fileparts(mfilename('fullpath'));
 
 % Find directories containing herbert/horace_init.m
-hor_init_dir = fileparts( ...
-    find_file('horace_init.m', {fullfile(install_root, 'Horace')}) ...
-);
-her_init_dir = fileparts( ...
-    find_file('herbert_init.m', {fullfile(install_root, 'Herbert')}) ...
-);
-
-% Find and update horace_on
-horace_on_path = find_file('horace_on.m.template', {install_root});
-horace_on_contents = fileread(horace_on_path);
-new_hor_on_contents = replace(horace_on_contents, HORACE_ON_SUB_PATTERN, hor_init_dir);
-new_hor_on_contents = replace(new_hor_on_contents, HERBERT_ON_SUB_PATTERN, her_init_dir);
-
-% Find and update herbert_on
-herbert_on_path = find_file('herbert_on.m.template', {install_root});
-herbert_on_contents = fileread(herbert_on_path);
-new_her_on_contents = replace(herbert_on_contents, HERBERT_ON_SUB_PATTERN, her_init_dir);
+% Do this first before find_userpath, so we can error out before potentially
+% creating a directory
+hor_init_dir = find_directory('horace_init.m', {fullfile(install_root, 'Horace')});
```

Comment:
why not pass `install_root` and `'Horace'` or `'Herbert'` arguments to `find_directory`? e.g. 
```
xxx_init_dir = find_directory('xxx_init.m', 'Xxxxx', install_root);
```

These are the only uses of `find_directory` so the `candidate_dirs` argument only ever contains one path